docs/_docs/editor/setup.md

```diff
@@ -19,8 +19,8 @@ supported and stability is not guaranteed.
 
 ## Quick Install
 
-Assuming you have met all the prerequisites for your platform, culminating with **Atom 1.7.4 or
-greater** being installed, the easiest way to install Nuclide is either at the command line:
+Assuming you have met all the prerequisites for your platform, culminating with *Atom 1.7.4 or
```

Comment:
Oh, we should totally fix these. The version we currently support is here: https://github.com/facebook/nuclide/blob/master/package.json#L114